lib/macho/load_commands.rb

```diff
@@ -204,15 +97,30 @@ def to_s
     # pretend that strings stored in LCs are immediately available without
     # explicit operations on the raw Mach-O data.
     class LCStr
+      # @return [Fixnum] the number of null bytes used to pad the string
+      attr_reader :npad
```

Comment:
Performance isn't really my concern here. My concern is that the padding inside a load command has nothing to do with individual strings represented by `LCStr`. The padding is something that is applied to the load command as a whole to make its size a multiple of either 4 or 8. `LCStr` simply isn't the right place to deal with this padding requirement.